### ISSUE TITLE
UI Home link should meet WCAG 2.1 AA color contrast requirements

### ISSUE THREAD

**Issue by nicoleslaw** [cagov/ui-claim-tracker#466]
### Description

Per CA's accessibility team, the gray link to `UI Home` is too light and does not meet color contrast ratio requirements for WCAG 2.1 AA.

Valery recommended `#6e7276`. Figma shows a darker `#555555` but the computed value is lighter (`rgba(0,0,0,.5)`).

### Acceptance Criteria

- [ ] Recheck nav gray
- [ ] Update to meet contrast ratio requirements

**Comment by dianagriffin** [comment]
@nicoleslaw @lomky The UI Home link actually gets replaced by the global header, correct? Should we close this ticket given the global header is in progress, or should we try to push this fix prior to the global header release (which doesn't have a set date yet)?

**Comment by nicoleslaw** [comment]
Great question—yes! The global header includes a link to UI Home but we can make sure the link text meets color contrast ratios on that ticket. #504 cc @nikkibrand
